migration/1687383705794-AddOptimismGoerliTokens.ts

```diff
@@ -0,0 +1,57 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+import { Token } from '../src/entities/token';
+import seedTokens from './data/seedTokens';
+import { NETWORK_IDS } from '../src/provider';
+
+export class AddOptimismGoerliTokens1687383705794
+  implements MigrationInterface
+{
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.manager.save(
+      Token,
+      seedTokens
+        .filter(token => token.networkId === NETWORK_IDS.OPTIMISM_GOERLI)
+        .map(t => {
+          t.address = t.address?.toLowerCase();
+          return t;
+        }),
+    );
```

Comment:
I think you need to add a condition to not add these test tokens in production env